composer.json

```diff
@@ -8,11 +8,17 @@
       "email": "contact@vinaikopp.com",
       "homepage": "http://vinaikopp.com",
       "role": "Developer"
+    },
+    {
+      "name": "Simone Marcato",
+      "email": "info@hevelop.com",
+      "homepage": "https://hevelop.com",
+      "role": "CTO"
```

Comment:
Please change your role to `Contributor` as that more accurately describes your role in the project (that is, this module).